.github/CODEOWNERS

```diff
@@ -0,0 +1,40 @@
+# https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+*       @ab506church/web
```

Comment:
This value is invalid, please make it match what is in the other file.

Reply:
done